packages/devextreme/testing/testcafe/tests/scheduler/appointments/allDay/allDay.ts

```diff
@@ -86,3 +86,53 @@ test('it should skip weekend days in timelineWorkWeek', async (t) => {
     },
   );
 });
+
+test('should work correctly for unsorted dataSource', async (t) => {
+  const {
+    takeScreenshot,
+    compareResults,
+  } = createScreenshotsComparer(t);
+
+  await t
+    .expect(await takeScreenshot('allDay-unsorted-datasource.png'))
+    .ok();
```

Comment:
Let's screenshot only a workspace, e.g:

```typescript
const SCHEDULER_SELECTOR = '#container';

test('...', async (t) => {
  const scheduler = new Scheduler(SCHEDULER_SELECTOR);
  
  // other test code.
  
  await takeScreenshot('allDay-unsorted-datasource.png', scheduler.workSpace);
  
  // other test code.
});
```

It helps us don't update this etalon when something outside the scheduler's workspace will change.